sections/06_discussion.md

```diff
@@ -80,8 +80,14 @@ training instances can be just as problematic.  In genomics, for example,
 labeled data may be derived from an experimental assay that has known and
 unknown technical artifacts, biases, and error profiles.  It is possible to
 weight training examples or construct Bayesian models to account for uncertainty
-in the data, but this is rarely done in practice. `TODO: @cgreene will edit
-this last line; it might be done in practice but not highlighted in papers`
+or non-independence in the data. For example, Park et al.
+[@doi:10.1371/journal.pcbi.1002957] estimated shared non-biological signal
```

Comment:
From my read, I think it doesn't matter too much. It's the Bayesian bit of that paper that's corrected for non-independence.